### PR TITLE
Update kubeadm-init.md

### PR DESCRIPTION
**Pull request by rdxmb**
fix anchor

### CODE REVIEW

File: content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md [hunk 1]
```diff
@@ -125,7 +125,7 @@ If your configuration is not using the latest version it is **recommended** that
 the [kubeadm config migrate](/docs/reference/setup-tools/kubeadm/kubeadm-config/) command.
 
 For more information on the fields and usage of the configuration you can navigate to our API reference
-page and pick a version from [the list](https://godoc.org/k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm#pkg-subdirectories).
+page and pick a version from [the list](https://godoc.org/k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm#section-subdirectories).
```
Comment (neolit123) [inline content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md:128]:
```suggestion
page and pick a version from [the list](https://pkg.go.dev/k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm#section-directories).
```


it's https://pkg.go.dev/k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm#section-directories like you said on https://github.com/golang/go/issues/45210

please keep the commits squashed to 1.

/sig cluster-lifecycle

Comment (rdxmb) [reply content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md:128]: seems it's too late for today... Will fix this tomorrow ...

Comment (rdxmb) [reply content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md]: I just approved that changes via GitHub. Is there no way to squash via the GUI like it is in GitLab?

Comment (neolit123) [reply content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md]: there is a bot command of sorts, the appovers know about.
one caveat is that it nukes GPG signatures on commits.

**Comment by RA489** [comment]
@rdxmb please squash the commit to one!

**Comment by rdxmb** [comment]
> @rdxmb please squash the commit to one!

can this be done here on Github?

**Comment by tengqm** [comment]
This is not an ideal solution. We'd consider publishing the "unpublished" kubeadm config API somehow, just like we are doing for kubelet, kube-scheduler, etc.
/approve

**Comment by sftim** [comment]
LGTM, will let SIG Cluster Lifecycle add /lgtm

**Pull request merged**